src/wallet/wallet.cpp

```diff
@@ -35,9 +35,11 @@
 #include <boost/algorithm/string/replace.hpp>
 
 static std::vector<CWallet*> vpwallets;
```

Comment:
Does it compile if you add a `GUARDED_BY` here?

Reply:
Good point @MarcoFalke!

Please add `GUARDED_BY(cs_wallets)` and verify by building with `--enable-werror` using  `clang`.

Reply:
Done.